### PR TITLE
feat: P2022-93 Configure i18next (translations)

### PR DESCRIPTION
**Pull request by yusts**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,5642 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (alexsimkovich) [inline yarn.lock]: PLease remove this file and add it to`.gitingore` file

File: public/locales/en/common.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (alexsimkovich) [inline public/locales/en/common.json:1]: Let's assume that `common.json` file will have common translations which are repeating on each page (footer, header, etc.). 
Also please create something meaningful, not just `p` :)
Please format json file properly

**Review by alexsimkovich — CHANGES_REQUESTED** [review]

**Comment by alexsimkovich** [comment]
@yusts in general, looks good.

**Comment by yusts** [comment]
@alexsimkovich @lukasz-galka 
Thanks for your comments. I already pushed the updated version.

**Review by lukasz-galka — APPROVED** [review]

**Pull request merged**